npc/re/quests/quests_dungeons_200.txt

```diff
@@ -0,0 +1,834 @@
+			setquest 8640;
+			close;
+		case 2:
+			end;
+		}
+		end;
+	case 1:
+	case 2:
+		mes "[Milla]";
+		mes "Yes, thank you for coming back.";
+		mes "Ha-am...";
+		mes "For me and my neighbors... for a good night's sleep.";
+		next;
+		if (select( "Listen to the request.", "Reject." ) == 2) {
+			mes "[Milla]";
+			mes "You can't do this if you're busy...";
+			mes "Ah, after all, the living do not hear that sound. Is this my agony only known to the dead...";
+			npctalk "Another Dead: Noisy! Get some sleep!", "#lv200althema", bc_self;
+			npctalk "Another Dead: He really can't sleep!", "#lv200althema2", bc_self;
+			close;
+		}
+		mes "[Milla]";
+		mes "Beloved living, thank you for your mercy.";
+		mes "It's a luxurious facility with a banquet hall and an opera house, but each one is a little different.";
+		mes "I'll ask for that kind of skill based on your kindness...";
+		next;
+		switch( select( "He looks cute", "That's not an instrument", "Plays the piano", "Guys with messy hair", "The ones with dolls", "Wake them up", "Stop" ) ) {
+		case 1:
+			switch( checkquest(8642,PLAYTIME) ) {
+			case -1:
+				break;
+			case 0:
+			case 1:
+				mes "[Milla]";
+				mes "I think they've had enough home today.";
+				mes "I hope you can help me tomorrow if you have time.";
+				close;
+			case 2:
+				erasequest 8642;
+				break;
+			}
+			switch( checkquest(8641,HUNTING) ) {
+			case -1:
+				mes "[Milla]";
+				mes "They look cute... they're terribly noisy.";
+				mes "^0000FF" + getmonsterinfo("GHOST_CUBE", MOB_NAME) + "^000000,";
+				mes "^0000FF" + getmonsterinfo("LUDE_GAL", MOB_NAME) + "^000000";
+				next;
+				mes "[Milla]";
+				mes "I wish I could beat 15 " + getmonsterinfo("GHOST_CUBE", MOB_NAME) + "";
+				mes "and 15 of their mate " + getmonsterinfo("LUDE_GAL", MOB_NAME) + ", who haunted the banquet hall of the dead,";
+				mes "and make them holy for a day.";
+				next;
+				if (select( "I accept", "I'm busy today" ) == 2) {
+					mes "[Milla]";
+					mes "Ummm...";
+					mes "Yeah... Well, I don't die just because I can't sleep...";
+					mes "I'm already dead";
+					close;
+				}
+				mes "[Milla]";
+				mes "Thank you for your kind heart.";
+				mes "Blessed be the mercy of the living!";
+				setquest 8641;
+				close;
+			case 0:
+			case 1:
+				mes "[Milla]";
+				mes "I wish I could beat 15 " + getmonsterinfo("GHOST_CUBE", MOB_NAME) + "";
+				mes "and 15 of their mate " + getmonsterinfo("LUDE_GAL", MOB_NAME) + ", who haunted the banquet hall of the dead,";
+				mes "and make them holy for a day.";
+				close;
+			case 2:
+				mes "[Milla]";
+				mes "Kya, did you hear them scream all the way here?";
+				mes "I think I can sleep a little deeper today thanks to you...";
+				mes "I have to make this commotion again tomorrow.";
+				mes "Thank you.";
+				erasequest 8641;
+				setquest 8642;
+				// getexp 15642570,10949790;	// pre-rebalance
+				getexp 20300909,10949790;	// rebalance (unknown jexp, copy from pre-rebalance)
```

Comment:
The jexp is always 70% of the base exp. This is true for mob experience as well. While there may be an odd exception for this particular quest, that's very unlikely.

Reply:
Ok, I'll calculate the jexp and leave a note about it, thanks!